mne_rsa/rsa.py

```diff
@@ -109,7 +109,7 @@ def _partial_correlation(rdm_data, rdm_model, masks=None, type="pearson"):
     return -R_partial[0, 1:]
 
 
-def rsa_gen(rdm_data_gen, rdm_model, metric="spearman", ignore_nan=False):
+def rsa_gen(rdm_data_gen, rdm_model, metric="spearman", ignore_nan=False, cache_ranks=True):
```

Comment:
I think it will be better to not have a `cache_ranks=True` parameter, but just always do it, unless `ignore_nan=True` is given. I can't think of a reason you would ever want to turn it off.